build/release-notes.md

```diff
@@ -8,6 +8,127 @@ toc_max: 2
 This page contains information about the new features, improvements, and bug
 fixes in [Docker Buildx](https://github.com/docker/buildx){:target="blank" rel="noopener" class=""}.
 
+## 0.10.2
+
+{% include release-date.html date="2023-01-30" %}
+
+> **Note**
+>
+> Buildx v0.10 enables support for a minimal [SLSA Provenance](https://slsa.dev/provenance/)
+> attestation, which requires support for [OCI-compliant](https://github.com/opencontainers/image-spec)
+> multi-platform images. This may introduce issues with registry and runtime support
+> (e.g. [Google Cloud Run and AWS Lambda](https://github.com/docker/buildx/issues/1533)).
```

Comment:
```suggestion
> Buildx v0.10 enables support for a minimal [SLSA Provenance](https://slsa.dev/provenance/){: target="_blank" rel="noopener" class="_" }
> attestation, which requires support for [OCI-compliant](https://github.com/opencontainers/image-spec){: target="_blank" rel="noopener" class="_" }
> multi-platform images. This may introduce issues with registries and runtimes that
> aren't OCI compliant, such as [Google Cloud Run and AWS Lambda](https://github.com/docker/buildx/issues/1533){: target="_blank" rel="noopener" class="_" }).
```

Reply:
Make this an include and reuse it